internal/convenience/repository/input_repository.go

```diff
@@ -390,6 +413,7 @@ func parseInput(res *sqlx.Rows) (*model.AdvanceInput, error) {
 	input.Payload = common.Hex2Bytes(payload)
 	input.MsgSender = common.HexToAddress(msgSender)
 	input.BlockTimestamp = time.UnixMilli(blockTimestamp)
+	input.EspressoBlockTimestamp = time.UnixMilli(espressoBlockTimestamp)
 	input.PrevRandao = prevRandao
```

Comment:
curious whether we could remove the var `prevRandao` and directly use `&input.PrevRandao` when `res.Scan`

Reply:
It's just to keep the style. I see no problem.